pages/servers.tsx

```diff
@@ -167,8 +215,8 @@ export async function getServerSideProps() {
     props: {
       // servers,
       filterList: {
-        topic: _orderBy(topic, "servers_count", "desc"),
-        language: _orderBy(language, "servers_count", "desc"),
```

Comment:
unless there's a `sort_by` param on the api (i couldn't find it), we can do sorting solely in the render function, which should sort whatever `getserversideprops` generates too